src/open_company_web/components/icon.cljs

```diff
@@ -0,0 +1,28 @@
+(ns open-company-web.components.icon
+  (:require [om-tools.dom :as dom]
+            [open-company-web.local-settings :as ls]))
+
+(defn icon
+  "Render an icon from our Nucleo icon set.
+
+  The icon set should be saved at resources/public/img/oc-icons.svg.
+  When exporting the icons from Nucleo make sure to use 'Export as <symbol>'.
+  Screenshot of Download window: http://i.imgur.com/ltAj6X1.png
+
+  Optional second argument can be used to pass a map of options:
+  - :accent-color (oc-blue)
+  - :size (30px)
+  - :stroke (2px)"
+  ([id] (icon id {}))
+  ([id {:keys [accent-color size stroke] :as opts}]
+   (assert id "Icon ID is required")
+   (let [outline-color "#0f0f0f"
+         accent-color  (or accent-color "#007A9D") ; oc-blue from scss/modules/_colors.scss
```

Comment:
@martinklepsch we should import this color from `open-company-web.oc-colors`, you can get it with
```clojure
(open-company-web.oc-colors/get-color-by-kw :oc-blue-light)
```